src/app.js

```diff
@@ -170,6 +231,38 @@ app.on('ready', () => {
     }
   }
 
+  function setNotificationTime(event, time) {
+    notificationWindow.close();
+    if (time && time !== store.get('notification.time')) {
+      let hours = time.split(':')[0];
+      let minutes = time.split(':')[1];
+      store.set('notification.time', time);
+      store.set('notification.hours', hours);
+      store.set('notification.minutes', minutes);
+      log.info(`Store updated - time=${time} hours=${hours} minutes=${minutes}`);
+      job.setTime(new CronTime(`0 ${minutes} ${hours} * * *`));
+    }
+  }
+
+  const job = new CronJob({
+    cronTime: `0 0 20 * * *`,
+    onTick: async function() {
```

Comment:
We can rewrite this function header as `async onTick() {` since we have babel goodness to match airbnb style guide

Reply:
I'm sorry I don't follow you here, can you provide me an example ?
I'm suppose to write : 
```js
onTick: async onTick() {}
```
What's the point with Babel and Airbnb style guide ?

Reply:
Right, the point is generally to keep a consistent code style across the project. For this project, the designated one chosen is the [airbnb style guide](https://github.com/airbnb/javascript). Its _mostly_ a preference thing but a lot of people really like Airbnb's specifications. 

Now we use Babel to _transpile_ our code to older more commonly supported versions of javascript so that we can write fancy things using newer versions of JS while still making sure all platforms understand our code. 

To follow the Airbnb style guide we need to follow this rule [object-shorthand](https://eslint.org/docs/rules/object-shorthand)

so instead of what you have, we can write: 

```
const job = new CronJob({
    cronTime: `0 0 20 * * *`,
    async onTick() {
      const data = await contribution(store.get('username'));
      if (data.currentStreak === 0 && Notification.isSupported()) {
        new Notification({
          title: 'Streaker',
          body: 'You didn\'t contribute today',
          icon: icon.icon
        }).show()
      }
    }
  });
```

Its a small change which keeps that one line of code simpler.

And because we have babel, it will probably get compiled down to what you wrote originally anyway 😛

If you run `npm run test` in the root dir it will give you a list of the all style errors you have. You can also probably configure vscode to do it automatically.

Sorry this was a long winded answer to a simple question... hope this cleared at least something up.

Reply:
Thank's a lot for the explanation !

I knew about Airbnb style guide and the utility of Babel but i've never used Babel before and Airbnb style guide too (Used StandardJS)

I will do that ! Thank's again for being kind with a noob like me